cmd/topolvm-controller/app/root.go

```diff
@@ -25,6 +25,9 @@ const (
 	// Derived from the usual 4096K blocks, 1024 inode default and journaling overhead,
 	// Allows for more than 80% free space after formatting, anything lower significantly reduces this percentage.
 	DefaultMinimumAllocationSizeExt4 = "32Mi"
+	// DefaultMinimumAllocationSizeBtrfs is the default minimum size for a filesystem volume with btrfs formatting.
+	// Btrfs minimum data block size is 4KiB, and by default, the metadata block size is 16 KiB.
```

Comment:
There is a difference between the comment and the default value. How about deleting the comment or describing it as an experimental result?

Reply:
changed as suggested, I liked that idea.